### ISSUE TITLE
Simplify opting out of license

### ISSUE THREAD

**Issue by RealRaven2000** [RealRaven2000/quickFilters#176]
See issue #167 - this was partially implemented (unblocking the assistant button by turning it into a menu which is always accessible regardless of license status). However it would be nice to have something that makes removal of a (expired) license easier. At the moment you have to delete the license from the text field manually and click the [validate] button, which isn't intuitive at all.

This could be a simple as a button [remove license] which can be followed by an opt in message box:

"If your license is expired you can remove it from quickFilters to return the Add-on to its initial behavior. Are you sure you want to remove the license?

You can always enter it again to be eligible for renewal or buy a new one later."

[Ok] [Cancel]
